pkg/workload/workload.go

```diff
@@ -294,7 +298,7 @@ func dropExcludedResources(input corev1.ResourceList, excludedPrefixes []string)
 	return res
 }
 
-func (i *Info) LocalQueueUsage(ctx context.Context, c client.Client, resWeights map[corev1.ResourceName]float64) (float64, error) {
+func (i *Info) GetLocalQueueUsage(ctx context.Context, c client.Client, resWeights map[corev1.ResourceName]float64) (float64, error) {
```

Comment:
Using Get prefix just to avoid conflict with LocalQueueUsage is a code smell. Let's try to unexport LocalQueueUsage. Or maybe the name `GetLocalQueueUsage` is wrong

Reply:
Addressed